keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/Action.kt

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.keel.api.ec2
+
+data class Action(
```

Comment:
`TargetGroupAction`?

Reply:
Yeah, I'd like to move these classes into packages and maybe make some inner classes where it clarifies things.